common/djangoapps/django_sudo_helpers/settings.py

```diff
@@ -0,0 +1,8 @@
+"""
+Settings for the django-sudo module.
+"""
+
+
+def apply_settings(django_settings):
+    """Set provider-independent settings."""
+    django_settings.SOCIAL_AUTH_PIPELINE += ('django_sudo_helpers.pipeline.create_sudo_session',)
```

Comment:
@bradenmacdonald First i did this in ```third_party_auth/settings.py``` but @cpennington asked me if we can move it into ```django_sudo_helpers``` and inject this into ```SOCIAL_AUTH_PIPELINE``` on startup.

I'll add a comment explaining this.